Readme.md

```diff
@@ -495,6 +509,71 @@ webfont: {
 }
 ```
 
+#### Custom Outputs
+
+```javascript
+webfont: {
+	icons: {
+		src: 'icons/*.svg',
+		dest: 'build/fonts',
+		options: {
+			customOutputs: [{
+				template: 'templates/icon-glyph-list-boilerplate.js',
+				dest: 'build/js/icon-glyph-list.js'
+			}, {
+				template: 'templates/icon-glyph-config-boilerplate.json',
+				dest: 'build/js/icon-glyphs.json'
+			}, {
+				template: 'templates/icon-web-home.html',
+				dest: 'build/',
+				context: {
+					testHeading: 'Hello, world!'
+				}
+			}]
+		}
+	}
+}
+```
+
+We might then include the following corresponding templates.
+
+The first, for `icon-glyph-list-boilerplate.js`, a file that outputs a list of icon-glyph slugs.
+
+```
+// file: icon-glyph-list-boilerplate.js
+
+(function() {
```

Comment:
Not very useful example because the local variable can’t be accessed from anywhere.